zebra-state/src/service/finalized_state/disk_format/tests/snapshot.rs

```diff
@@ -124,12 +126,12 @@ fn snapshot_raw_rocksdb_column_family_data(db: &DiskDb, original_cf_names: &[Str
             .expect("RocksDB API provides correct names");
 
         // Correctness: Multi-key iteration causes hangs in concurrent code, but seems ok in tests.
-        let mut cf_iter = db.full_iterator_cf(&cf_handle, rocksdb::IteratorMode::Start);
+        let cf_iter = db.full_iterator_cf(&cf_handle, rocksdb::IteratorMode::Start);
 
         // The default raw data serialization is very verbose, so we hex-encode the bytes.
         let cf_data: Vec<KV> = cf_iter
-            .by_ref()
-            .map(|(key, value)| KV::new(key, value))
+            .map_ok(|(key, value)| KV::new(key, value))
+            .filter_map(|kv| kv.ok())
```

Comment:
We can't ignore database errors in the tests, we can panic and fail the tests instead.